scripts/publish-gh-pages.sh

```diff
@@ -20,7 +20,8 @@ then
 fi
 
 echo overriding new yaml / chart files from master branch
-git checkout origin/master -- snyk-monitor snyk-monitor-cluster-permissions.yaml snyk-monitor-deployment.yaml snyk-monitor-namespaced-permissions.yaml README.md
+git checkout origin/master -- snyk-monitor snyk-monitor-cluster-permissions.yaml snyk-monitor-deployment.yaml snyk-monitor-namespaced-permissions.yaml
+git reset ./snyk-monitor/README.md
```

Comment:
Why do we need to reset?

We could delete the READMEs manually from gh-pages if the `reset` just tries to undo the files